roles/jenkins_job_launcher/README.md

```diff
@@ -0,0 +1,26 @@
+# jenkins_job_launcher
+
+jenkins_job_launcher role triggers Jenkins job using the uri module.
+the role makes use of username + user API token to authenticate with Jenkins.
+
+Role Variables
+--------------
+
+Required variables are: 
```

Comment:
Could this be in a nice table like [this](https://github.com/redhatci/ansible-collection-redhatci-ocp/tree/main/roles/create_pr)?